# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1365967097 -7200
#      Sun Apr 14 21:18:17 2013 +0200
# Node ID b348c5835ce4b3eec12b7cbda59c2bb3965f6185
# Parent  7af39cc909822e7c93ce2e9ced6215d6ca21f4de
* file-editor.cc(construct): icon search.png instead of find.png

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -688,17 +688,17 @@ file_editor::construct ()
                    tr ("Toggle &breakpoint"), _tool_bar);
   QAction *remove_all_breakpoints_action
     = new QAction (QIcon (":/actions/icons/bp_rm_all.png"),
                    tr ("&Remove All breakpoints"), _tool_bar);
 
   QAction *comment_selection_action   = new QAction (tr ("&Comment Selected Text"),_tool_bar);
   QAction *uncomment_selection_action = new QAction (tr ("&Uncomment Selected Text"),_tool_bar);
 
-  QAction *find_action = new QAction (QIcon(":/actions/icons/find.png"),
+  QAction *find_action = new QAction (QIcon(":/actions/icons/search.png"),
                                       tr ("&Find and Replace"), _tool_bar);
 
   _run_action = new QAction (QIcon(":/actions/icons/artsbuilderexecute.png"),
                              tr("Save File And Run"), _tool_bar);
 
   QAction *goto_line_action  = new QAction (tr ("Go&to Line"), _tool_bar);
 
   // the mru-list and an empty array of actions
